.mypy.ini

```diff
@@ -69,6 +69,10 @@ disallow_any_expr = false
 # crashes with some decorators like `@functools.cache`:
 disallow_any_expr = false
 
+[mypy-awx_plugins.credentials.azure_mi]
+# crashes with some decorators like `@functools.cache`:
```

Comment:
Unless, there's an actual error violation that's happening, let's not disable rules globally. And if there are, it's best to inline ignores on specific lines, with explanatory comments.